src/main/java/com/auth0/AuthorizeUrl.java

```diff
@@ -119,8 +167,41 @@ public String build() throws IllegalStateException {
         if (used) {
             throw new IllegalStateException("The AuthorizeUrl instance must not be reused.");
         }
+
+        if (response != null) {
+            TransientCookieStore.SameSite sameSiteValue = containsFormPost() ?
+                    TransientCookieStore.SameSite.NONE : TransientCookieStore.SameSite.LAX;
+
+            // Also store in Session just in case developer uses deprecated
+            // AuthenticationController.handle(HttpServletRequest) API
+            if (state != null) {
```

Comment:
Do we want to have all these null checks? What about making them nullable on the storeState / setState methods? Just a thought.

Reply:
If it doesn't add too much complexity to the storage methods (I don't think it will), that's probably a good change to make. Will look into.

Reply:
Updated to move null value handling out of `AuthorizeUrl` and into store handling.